project/Dependencies.scala

```diff
@@ -65,6 +65,7 @@ object Dependencies {
   )
 
   private val sprayServerDependencies = List(
+    "org.webjars" % "swagger-ui" % "2.2.2" % Provided,
```

Comment:
Whoops! Ignore that part. It's an old issue that doesn't need to be addressed in this PR.

Instead, the swagger version info needs to be refactored. Instead of being written separately in two places, the swagger version should be written by something like [`writeVersionConf`](https://github.com/broadinstitute/cromwell/blob/e3bb43f6ca51ca6ee6c4260f86c5760059719ec1/project/Version.scala#L22-L31), and then [`SwaggerService.swaggerUiVersion`](https://github.com/broadinstitute/cromwell/blob/e3bb43f6ca51ca6ee6c4260f86c5760059719ec1/engine/src/main/scala/cromwell/webservice/CromwellApiService.scala#L24) should be reading it from the [`cromwell-version.conf`](https://github.com/broadinstitute/cromwell/blob/e3bb43f6ca51ca6ee6c4260f86c5760059719ec1/engine/src/main/scala/cromwell/webservice/CromwellApiService.scala#L284).